index.html

```diff
@@ -9,15 +9,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" type="text/css" href="style.css">
-    <title>Portfolio</title>
+    <title>Shri Sakthi Portfolio Website</title>
 </head>
 <body>
 
     <div class="starting-page"> 
         <header >
             <nav id="navigator_bar">
                 <a class="logo" href="#"><h2>Mylogo</h2></a>
```

Comment:
Good job so far @Shri-sakthi-v , Just a small thing to mention, the color of the logo doesn't have enough contrast with the background kindly make it a bit darker for a better user experience